core-services/release-controller/_releases/priv/release-ocp-4.15.json

```diff
@@ -62,7 +62,7 @@
             "upgradeFromRelease": {
```

Comment:
@deepsm007 Hey sorry for the interrupt but I need some help here. Do you know if 
```json
"upgradeFromRelease": {
  "candidate": {
    "stream": "nightly",
     "version": "4.14"
 }
```
can be safely replaced by
```json
  "upgradeFrom": "PreviousMinor"
```
in this context? It would be easier for us to maintain as the won't have to bump the version every time we perform the branch cut.

Reply:
@bradmwilliams words from [this thread](https://redhat-internal.slack.com/archives/CNHC2DK2M/p1692023891223699):
> No, the release-controller would return an rc version always and not the corresponding nightly or ci stream.

So no, it doesn't work here.